app/models/mappers/check-activities.js

```diff
@@ -0,0 +1,43 @@
+const { formatToGds } = require('../../lib/date-helpers')
+
+const activityMapper = {
```

Comment:
I thought we concluded this mapper wasn't necessary. We were just going to construct the text as `${event.activity.activityLabel} ${event.activity.activityType}`

Reply:
Initially, then we discussed that in future we will need a mapper, so added it back in.

Reply:
It's just that we can't rely on a mapper if they have admin privs to edit the events and add new ones and delete etc - we have to rely on simply constructing the string based on a correct label in the DB

Reply:
removed